.github/workflows/rust-compile.yml

```diff
@@ -0,0 +1,72 @@
+on: [push, pull_request]
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
```

Comment:
You can add a rust-toolchain file to ensure stable rust version. This is important otherwise someone making a pr could suddenly get a failing pr.

If we also add dependabot, the version is automatically bumped.

Reply:
Added this. Maybe dependabot in a later commit, don't know how to set that up exactly.

Reply:
If you add this information it should work out of the box! Another PR is fine. :)

https://github.com/mun-lang/mun/blob/main/.github/dependabot.yml